src/main/java/com/bunq/sdk/http/BunqHeader.java

```diff
@@ -0,0 +1,100 @@
+package com.bunq.sdk.http;
+
+import java.util.Collection;
+import java.util.Map;
+
+public enum BunqHeader {
+    attachmentDescription("X-Bunq-Attachment-Description"),
```

Comment:
make the enum CAP_SNAKE_CASE please see: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names